collector.go

```diff
@@ -107,6 +112,31 @@ func newSpaceliftCollector(ctx context.Context, httpClient *http.Client, session
 			"The number of seats used in the current billing period",
 			nil,
 			nil),
+		currentStacksCountByState: prometheus.NewDesc(
+			"spacelift_current_stacks_count_by_state",
+			"The number of stacks by certain stacks",
```

Comment:
```suggestion
			"The number of stacks grouped by state",
```

Please also fix the readme 🙏🏻

Reply:
Can you also fix the readme please?